js/app/state/create.js

```diff
@@ -18,6 +18,10 @@ function(grid, generator, background, music){
         background.start(game);
         grid.display(game);
         generator.start(game);
+        score.init(game, grid, generator);
+        
+        game.scoreboard = score;
+        game.generator = generator;
```

Comment:
We should not be attaching anything to the game object. If a module needs access to the scoreboard or generator, it should depend on them explicitly, either by requiring them, or accepting them at construction/initialization.